scenario/constraints/ProposalConstraint.ts

```diff
@@ -75,7 +75,13 @@ export class ProposalConstraint<T extends CometContext> implements StaticConstra
             await executeOpenProposal(governanceDeploymentManager, proposal);
           }
           debug(`${label} Open proposal ${proposal.id} was executed`);
+        }
+        catch (err) {
```

Comment:
super nit, but maybe put the `catch` on the same line as the close bracket:

```
} catch (err) {
```

Reply:
Yeah that's more preferable for sure.